.travis.yml

```diff
@@ -37,8 +37,9 @@ matrix:
         - sudo pip install tox
      before_script:
         - export DISPLAY=:0.0
-        - Xvfb ${DISPLAY} -screen 0 800x600x16 & PID=$!
-        - sleep 3  # give xvfb some time to start
+        # set pixel depth https://docs.travis-ci.com/user/gui-and-headless-browsers/
+        - /sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_00.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :00 -ac -screen 0 1280x1024x16
```

Comment:
why this change is not applied to the osx job?

Reply:
The original is preferred as it's more simple. On OSX it worked at least earlier. Related https://github.com/swift-nav/piksi_tools/pull/974#discussion_r254592223

Reply:
ok

Reply:
Looks like latest master doesn't have this problem anymore, so I'll revert this change.